### PR TITLE
Migrate to Vue 3:

### PR DESCRIPTION
**Pull request by Tehsurfer**
Filtering options were removed as they are not used on sparc.science anymore

Note that this can be tested using

`npm install @abi-software/plotvuer@alpha`

### CODE REVIEW

**Review by alan-wu — CHANGES_REQUESTED** [review]
All the map svg icons are not displaying as they are commented out. The props for el-popover have been changed in element-plus so they need to be updated. Please update your pull request.

**Review by alan-wu — COMMENTED** [review]
With vite - export should be stated in the package.json file now - similar to the following:

```
  "type": "module",
  "main": "./dist/scaffoldvuer.umd.cjs",
  "module": "./dist/scaffoldvuer.js",
  "exports": {
    ".": {
      "import": "./dist/scaffoldvuer.js",
      "require": "./dist/scaffoldvuer.umd.cjs"
    },
    "./dist/style.css": "./dist/style.css",
    "./src/*": "./src/*"
  },
```

File: package.json [hunk 1]
```diff
@@ -17,33 +18,37 @@
...
   "dependencies": {
-    "@abi-software/svg-sprite": "^0.1.15",
+    "@abi-software/svg-sprite": "^0.4.0-vue3-beta.0",
+    "@element-plus/icons-vue": "^2.3.1",
     "core-js": "^3.7.0",
     "css-element-queries": "^1.2.3",
     "current-script-polyfill": "^1.0.0",
     "d3-time-format": "^3.0.0",
+    "element-plus": "^2.5.3",
     "element-ui": "^2.13.0",
```
[the remark this answers is not in the document]
Comment (Tehsurfer) [reply package.json:28]: @alan-wu Remove element-ui right?

[the remark this answers is not in the document]
Comment (alan-wu) [reply package.json:28]: Yes, element-ui. I somehow targeted the wrong line.

[the remark this answers is not in the document]
Comment (alan-wu) [reply package.json:28]: I did not target the wrong line, it is confusing with the green + colour.

**Review by alan-wu — APPROVED** [review]

**Pull request merged**